### PR TITLE
Upgrade `client-wasm` to `sdk-wasm`

### PR DESCRIPTION
**Pull request by PhilippGackstatter**
# Description of change

Upgrades `client-wasm` to `sdk-wasm`.

## Links to any relevant issues
Be sure to reference any related issues by adding `fixes issue #`.

## Type of change
Add an `x` to the boxes that are relevant to your changes.

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] Enhancement (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Fix

## How the change has been tested
Describe the tests that you ran to verify your changes.
Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist
Add an `x` to the boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by eike-hass** [comment]
We need to update the Readme in the wiki repo as well

File: bindings/wasm/examples/src/util.ts [hunk 1]
```diff
@@ -81,17 +91,17 @@ export async function ensureAddressHasFunds(client: Client, addressBech32: strin
...
         { hasTimelock: false },
         { hasStorageDepositReturn: false },
     ]);
-    const outputs = await client.getOutputs(outputIds);
+    const outputs = await client.getOutputs(outputIds.items);
 
     let totalAmount = 0;
     for (const output of outputs) {
-        totalAmount += Number(output.output.amount);
+        totalAmount += Number(output.output.getAmount());
```
Comment (abdulmth) [inline bindings/wasm/examples/src/util.ts:104]: We shouldn't convert a `bigint` to a `number`. Maybe return a `bigint` here instead, arithmetic works on `bigint` just like on number.

File: bindings/wasm/cypress/support/setup.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
-import * as client from "@iota/client-wasm/web";
+import init from "@iota/sdk-wasm/web";
```
Comment (abdulmth) [inline bindings/wasm/cypress/support/setup.js:1]: shouldn't this be `import { init } from ..` ? 🤔

**Review by abdulmth — APPROVED** [review]

File: bindings/wasm/cypress/support/setup.js (hunk 2 above)
Comment (PhilippGackstatter) [reply bindings/wasm/cypress/support/setup.js:1]: This is how it is done in https://github.com/iotaledger/iota-sdk/blob/develop/bindings/wasm/README.md#web, and it didn't work before when I used `import * as sdk ...` and then used `sdk.init`, so I'll not touch that again :joy:.

File: bindings/wasm/examples/src/util.ts (hunk 1 above)
Comment (PhilippGackstatter) [reply bindings/wasm/examples/src/util.ts:104]: Getting this issue though:
```
Operator '>' cannot be applied to types 'BigInt' and 'number'.ts(2365)
```

Comment (abdulmth) [reply bindings/wasm/examples/src/util.ts:104]: you have to convert the number to a bigint using `BigInt(variable)`

Comment (PhilippGackstatter) [reply bindings/wasm/examples/src/util.ts:104]: Yeah getting this error on a `BigInt(number)` value. It's not a conversion problem it's that you cannot use `>` on a bigint.

File: bindings/wasm/README.md [hunk 3]
```diff
@@ -290,7 +290,7 @@ client
 // or
 
 (async () => {
-  await client.init();
+  await sdk.init();
```
[the remark this answers is not in the document]
Comment (PhilippGackstatter) [reply bindings/wasm/README.md:293]: How do I import init and `* as sdk` in one line?

**Pull request merged**